core/src/test/scala/pureconfig/PureconfSuite.scala

```diff
@@ -592,6 +594,25 @@ class PureconfSuite extends FlatSpec with Matchers with OptionValues with TryVal
     config.toOption.value.url shouldBe new URL(expected)
   }
 
+  case class ConfWithUUID(uuid: UUID)
+
+  it should "be able to read a config with a UUID" in {
+    val expected = "d25aed6a-ef6d-4c10-954c-02edc949aef1"
+    val config = loadConfig[ConfWithUUID](ConfigValueFactory.fromMap(Map("uuid" -> expected).asJava).toConfig)
+    config.toOption.value.uuid shouldBe UUID.fromString(expected)
+  }
+
+  it should "round trip a UUID" in {
+    saveAndLoadIsIdentity(ConfWithUUID(UUID.randomUUID()))
```

Comment:
@derekmorr Eh, I wouldn't. Other folks might. I feel like this test covers that case. I kind of like having one simple case with fixed data in addition to my property tests.